android/app/src/main/java/app/organicmaps/sdk/Map.java

```diff
@@ -142,6 +146,8 @@ public void updateMyPositionRoutingOffset(int offsetY)
 
   public void onSurfaceCreated(final Context context, final Surface surface, Rect surfaceFrame, int surfaceDpi)
   {
+    assert mLocationHelper != null : "LocationHelper must be initialized before calling onSurfaceCreated";
```

Comment:
```suggestion
    if (mLocationHelper == null)
      throw new Exception("LocationHelper must be initialized before calling onSurfaceCreated");
```

Reply:
Why? assert is the same thing but with less code

Reply:
AFAIR, assertions on Andorid should be explicitly enabled to work with something like `adb shell setprop debug.assert 1`

Reply:
assert is ok here indicating to developer that ```setLocationHelper``` should be called first.
Logically, this param is NotNull